### PR TITLE
[RA2 Ch2 Ch4] AF_XDP specs

### PR DESCRIPTION
**Pull request by petorre**
Closes [#3354](https://github.com/anuket-project/anuket-specifications/issues/3354)

If needed, the long list of prerequisites is [here](https://github.com/intel/afxdp-plugins-for-kubernetes?tab=readme-ov-file#prerequisites).

### CODE REVIEW

**Review by CsatariGergely — APPROVED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 1]
```diff
@@ -167,6 +167,12 @@ the following specifications:
        the Kubernetes Cluster.
      - tbd
      - :cite:t:`anuket-ri2` Chapter 4, section Installation on Bare Metal Infratructure
+   * - ra2.ch.019
+     - AF_XDP Zero Copy capable netdev’s
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:171]:
```suggestion
     - AF_XDP Zero Copy capable netdevs
```

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 2]
```diff
@@ -167,6 +167,12 @@ the following specifications:
...
+     - AF_XDP Zero Copy capable netdev’s (dependent on AF_XDP Zero Copy NIC driver) must be available in Kubernetes
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:172]:
```suggestion
     - AF_XDP Zero Copy capable netdevs (dependent on AF_XDP Zero Copy NIC driver) must be available in a compliant Kubernetes cluster
```

**Review by rgstori — CHANGES_REQUESTED** [review]

**Comment by rgstori** [comment]
thanks @petorre , added just a couple of tweaks

**Comment by petorre** [comment]
> thanks @petorre , added just a couple of tweaks

Thanks. I committed the change, but changed "cluster" to "worker" as it is per worker node.

**Review by rgstori — APPROVED** [review]

**Pull request merged**